<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="320" />
    <param name="image_height" value="240" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="elp_camera" />
  </node>
  
  <!-- Запустить визуализацию -->
  <node name="rqt_image_view" pkg="rqt_image_view" type="rqt_image_view" args="/usb_cam/image_raw" />
  
  <node name="yuyv_to_rgb_converter" pkg="ros_elp" type="yuyv_to_rgb_converter.py" output="screen"/>
</launch>
